pahfit/base.py

```diff
@@ -110,7 +110,7 @@ class PAHFITBase:
         x_0, x_0_limits, x_0_fixed, fwhms, fwhms_limits, fwhm_fixed}.
     """
 
-    def __init__(self, obs_x, obs_y, param_info=None, filename=None, tformat=None):
+    def __init__(self, obs_x, obs_y, noguess, param_info=None, filename=None, tformat=None):
```

Comment:
This should be changed to a boolean keyword argument.  Or do you think it is a required argument and you want to the users to explicitly input it.

For a keyword argument, I would suggest it should be renamed to "estimatestart" or "estimateinnit" something like that.  And then have it only call the "guess" function if it is set to True.  This would mean that the default behavior is to not do anything and I think this makes more sense when thinking about instantiating a class.  At least in my brain.

Reply:
I was thinking it might be useful to set it as a user defined keyword argument, because people might want to completely bypass the guess function, for instance if re-fitting a spectrum and you want to use as a starting point the initial run results and not go through the guess function again. 

No preference on the naming. "estimatestart" or "estimateinit" seems good to me.

Reply:
Currently it is not a keyword.  But a required positional argument.  To set it as a keyword, it would have to be "noguess=True" for example.

Reply:
Sorry I was thinking about a positional argument, because currently there is no other "interaction" with the user other than providing arguments.